index.html

```diff
@@ -11,16 +11,28 @@
 </head>
 
 <body>
-  <header>
-    <p class="personal-logo"> Jose RM</p>
-    <nav class="hamburger-menu">
-      <nav class="burger-icon"><span style="color: #fff;"><i class="fa-solid fa-bars"></span></i></nav>
+  <header class="topnav nav">
+    <nav>
+      <div class="menu">
+        <img id="hamburger" src="./images/hamburger-icon.svg" alt="Display menu">
+      </div>
     </nav>
-    <nav class="nav-header">
-      <a href=""> Home</a>
-      <a href="">About</a>
-      <a href="">Contact</a>
-      <a href=""><img src="./images/mail-icon.png" alt="Email Access Icon"></a>
+    <div class="menu-display">
+      <div class="xicon"><img src="./images/x-icon.png" alt="Close menu"></div>
+      <ul id="items-menu">
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </div>
```

Comment:
- [ ] Please make sure that you list the navigation links on the mobile menu so that someone can navigate your portfolio to all sections of the page. As you can see below there is no list of `Portfolio`, `About`, or `Contact` in your project mobile menu.
![Screenshot from 2023-02-08 15-41-01](https://user-images.githubusercontent.com/92303891/217577864-67265585-4911-4bc4-899b-2d26297da51e.png)

Reply:
Thanks Desmond, I will do.